src/main/java/com/ibm/watson/modelmesh/ModelMeshApi.java

```diff
@@ -829,15 +829,15 @@ public void onHalfClose() {
              * @param status null for requests, non-null for responses
              * @param takeOwnership whether the processor should take ownership
              */
-            private void processPayload(ByteBuf data, String payloadId, String modelId, String methodName,
+            private void processPayload(ByteBuf data, String payloadId, String vModelId, String modelId, String methodName,
```

Comment:
@njhill -- I noticed you added the `vModelId` param **_before_** the `modelId`, although all the method invocations have the `vModelId` parameter **_after_** the `resolvedModelId`.

I swapped them in my rebase/merge conflict resolution commit:

https://github.com/kserve/modelmesh/pull/123/files#diff-553ec9c8cb68586417339d2c370b4cea48d6def75ab451084aff24d9c75ac404R833

<img width="947" alt="image" src="https://github.com/kserve/modelmesh/assets/12246093/4399df4b-df74-4a29-854c-2ea929b1cba0">

And I added the `vModelId` parameter to the JavaDoc

Reply:
nice spot @ckadner :)

Reply:
The changes looks overall, but I have the same concern about the method signature changes.
Is there a way to make sure this would not affect the behavior of this method's callers?